apps/nextjs-example/pages/index.tsx

```diff
@@ -8,6 +8,8 @@ import dynamic from "next/dynamic";
 import Image from "next/image";
 import { useAutoConnect } from "../components/AutoConnectProvider";
 
+import "regenerator-runtime";
```

Comment:
what is this for..?

Reply:
hey ya, forgot to mention in the pr. basically it's adding async runtime support for those browsers that doesn't have async support out of the box.

Reply:
I see. why is it needed here...? is this needed to support your wallet?

Reply:
yep. our plugin currently use async func for simplicity... 
thought maybe we can add some browser support for those users who use earlier version of browsers, just in case...

Reply:
and it def can be extracted into another separate pr. let me know if that's the way you prefer

Reply:
yes lets remove it from this PR. 

The adapter itself uses async functions and we haven't had any issues with that, so dont think it is needed.

Reply:
✅ just removed that pkg. should be cleaner now.